pkg/plugins/manager/loader/loader.go

```diff
@@ -271,7 +271,12 @@ func setDefaultNavURL(p *plugins.Plugin) {
 			p.DefaultNavURL = path.Join("/plugins/", p.ID, "/page/", include.Slug)
 		}
 		if include.Type == "dashboard" {
-			p.DefaultNavURL = path.Join("/dashboard/db/", include.Slug)
+			if include.UID == "" {
```

Comment:
Want to check if `include.DashboardURLPath()` is empty like in index.go? Then maybe we need to change the message?